test/timeout-fs.spec.ts

```diff
@@ -1,62 +1,66 @@
 import * as Promise from 'bluebird';
 import {expect} from 'chai';
 import {FileSystem} from '../src/api';
-import {assertFileSystemContract, ignoredDir, ignoredFile} from './implementation-suite'
+import {
+    assertFileSystemContract,
+    ignoredDir,
+    ignoredFile,
+    dirName,
+    fileName,
+    content
+} from './implementation-suite'
 import {SlowFs} from '../test-kit/drivers/slow-fs';
 import {MemoryFileSystem} from '../src/memory-fs';
 import {TimeoutFs} from '../src/timeout-fs';
 
-describe('the timeout file system imeplementation', ()=>{
+describe('the timeout file system imeplementation', () => {
     const timeout = 200;
     const accuracyFactor = 0.9;
-    assertFileSystemContract(() => 
-    Promise.resolve(new TimeoutFs(timeout , new MemoryFileSystem(undefined, [ignoredDir, ignoredFile]))), 
-    {interval:1, noExtraEventsGrace:10, timeout:30});
 
-    describe(`delayed timeout test`, () => {
+    assertFileSystemContract(() =>
+        Promise.resolve(new TimeoutFs(timeout , new MemoryFileSystem(undefined, [ignoredDir, ignoredFile]))),
+        {interval:1, noExtraEventsGrace:10, timeout:30}
+    );
 
-         let fs: FileSystem;
-         let startTimestamp: number;
-         let endTimestamp: number;
-         const dirName = 'dir';
```

Comment:
good
this code was copy-pasted from 
https://github.com/wix/kissfs/blob/master/test-kit/test/slow-fs.spec.ts
so it needs the same treatment